scrapy/contrib/loader/processor.py

```diff
@@ -63,6 +63,33 @@ def __call__(self, values):
         return values
 
 
+class JmesProcessor(object):
+    """
+        Query the input string for the jmespath (given at instantiation), and return the answer
+        Requires : jmespath(https://github.com/boto/jmespath), ast
+        Note: JmesProcessor accepts only a single string at a time and returns string/dict/None based on the jmespath query.
+    """
```

Comment:
Thanks! 
The Sphinx link I posted here is not relevant, but it is relevant for `__call__` docstring.